workflow/executor.py

```diff
@@ -382,11 +382,15 @@ def _load_source(self, logger: logging.Logger, env_config: EnvConfig, rai_config
                 for d in src["dates"]:
                     logger.info(f"Loading partition for date {d['date']}")
 
-                    resources = d["resources"]
-                    self._load_resource(logger, env_config, rai_config, resources, src)
+                    for res in d["resources"]:
+                        self._load_resource(logger, env_config, rai_config, [res], src)
         else:
             logger.info(f"Loading source '{source_name}' not partitioned by date ")
-            self._load_resource(logger, env_config, rai_config, src["resources"], src)
+            if self.collapse_partitions_on_load:
```

Comment:
Could you please add logging statements similar those on lines 373 and 381? Feel free to make them debug rather than info.

Reply:
added